Makefile

```diff
@@ -57,47 +57,52 @@ pip-compile:
 # Build #
 #########
 
-## docker-build:                builds the docker container for the pipeline
-.PHONY: docker-build
-docker-build:
-	BUILD_TYPE="" PIP_VERSION=${PIP_VERSION} PIPELINE_FAMILY=${PIPELINE_FAMILY} ./scripts/docker-build.sh
-
 ## generate-api:                generates the FastAPI python APIs from notebooks
 .PHONY: generate-api
 generate-api:
 	PYTHONPATH=. unstructured_api_tools convert-pipeline-notebooks \
 		--input-directory ./pipeline-notebooks \
 		--output-directory ./${PACKAGE_NAME}/api
 
+
 #########
 # Local #
-########
-
-## run-notebooks-local:         runs the container as a docker compose file
-.PHONY: run-notebooks-local
-run-notebooks-local:
-	docker-compose -p ${PIPELINE_FAMILY} -f docker/docker-compose-notebook.yaml up
-
-## stop-notebooks-local:        stops the container
-.PHONY: stop-notebooks-local
-stop-notebooks-local:
-	docker-compose -p ${PIPELINE_FAMILY} stop
-
-## start-app-local:             runs FastAPI in the container with hot reloading
-.PHONY: start-app-local
-start-app-local:
-	docker-compose -p ${PIPELINE_FAMILY}-api -f docker/docker-compose-api.yaml up
-
-## stop-app-local:              stops the container
-.PHONY: stop-app-local
-stop-app-local:
-	docker-compose -p ${PIPELINE_FAMILY}-api stop
-
-## run-app-dev:                 runs the FastAPI api with hot reloading
-.PHONY: run-app-dev
-run-app-dev:
+#########
+
+## run-jupyter:                 starts jupyter notebook
+.PHONY: run-jupyter
+run-jupyter:
+	PYTHONPATH=$(realpath .) JUPYTER_PATH=$(realpath .) jupyter-notebook  --no-browser --NotebookApp.token='' --NotebookApp.password=''
```

Comment:
I think the most common use case would be that along with running the server, the user would also want a browser window open to the Jupyter file explorer. With that in mind could we drop the --no-browser flag?